datadogclient/formatter.go

```diff
@@ -23,6 +23,12 @@ func (f Formatter) Format(prefix string, maxPostBytes uint32, data map[MetricKey
 	return result
 }
 
+func (f Formatter) FormatEvent(prefix string, maxPostBytes uint32, event Event) [][]byte {
+	var result [][]byte
+	eventsBytes, _ := json.Marshal(event)
```

Comment:
The error here is being ignored. Recommend instead that the error is returned to be handled by the caller.